cdap-docs/admin-manual/source/installation/mapr.rst

```diff
@@ -97,6 +115,29 @@ that value instead for ``/cdap/tx.snapshot``.
 .. |hdfs-user| replace:: ``cdap``
 
 .. include:: /../target/_includes/mapr-configuration.rst
+    :end-before:   .. _mapr-configuration-options-may-need
+
+#. Due to an issue with the version of the Kafka ZooKeeper client shipped with MapR, 
+   it is necessary to disable use of the embedded Kafka in CDAP by setting these properties:
```

Comment:
It might be nice, maybe below the properties, to explain the impact of this... that the container logs will not be streamed back to the master process log file.  It might also be nice to link to a bug for more info (if we have the actual kafka bug... everything i've seen is on our side).